Doc/Tutorial/chapter_entrez.tex

```diff
@@ -261,14 +263,13 @@ \section{ESummary: Retrieving summaries from primary IDs}
 \begin{verbatim}
 >>> from Bio import Entrez
 >>> Entrez.email = "A.N.Other@example.com"     # Always tell NCBI who you are
->>> handle = Entrez.esummary(db="journals", id="30367")
+>>> handle = Entrez.esummary(db="nlmcatalog", term="[journal]", id="101660833")
 >>> record = Entrez.read(handle)
->>> record[0]["Id"]
-'30367'
->>> record[0]["Title"]
-'Computational biology and chemistry'
->>> record[0]["Publisher"]
-'Pergamon,'
+>>> info = record[0]['TitleMainList'][0]
+>>> print("Journal info\nid: {}\nTitle: {}".format(record[0]["Id"], info["Title"]))
```

Comment:
Do you personally find this easier to understand as an example compared to the old version just looking at the field values directly?

(I'm still not used to the ``.format()`` which makes me prejudiced here)